app/ide-desktop/lib/dashboard/src/modals/TermsOfServiceModal.tsx

```diff
@@ -70,6 +71,16 @@ export function TermsOfServiceModal() {
   const isAccepted = localVersionHash != null
   const shouldDisplay = !(isAccepted && isLatest)
 
+  const formSchema = ariaComponents.Form.useFormSchema(schema =>
+    schema.object({
+      agree: schema
+        .boolean()
+        // we accept only true
+        .refine(value => value, getText('licenseAgreementCheckboxError')),
```

Comment:
prefer `schema.literal(true)`

Reply:
They're not equal. False is still a value, but, we want to show error when value is false.